cibuildwheel/linux.py

```diff
@@ -104,10 +105,12 @@ def build(options: BuildOptions) -> None:
         ('pp', 'manylinux_x86_64', options.manylinux_images['pypy_x86_64']),
     ]
 
-    if not os.path.realpath(options.package_dir).startswith(os.path.realpath('.')):
+    pwd = Path().resolve()
```

Comment:
I think `cwd` would be more appropriate for this variable - `pwd` means 'Print Working Directory'

Reply:
In fact [`Path.cwd()`](https://docs.python.org/3/library/pathlib.html#pathlib.Path.cwd) is part of the library

Reply:
Huh, good point. I always thought `pwd` was "present working directory". Guess I learned something today.